frontends/ol-components/src/components/Input/Input.tsx

```diff
@@ -52,10 +52,10 @@ const Input = styled(InputBase)(({
       paddingRight: "12px",
       borderRadius: "4px",
       "&.MuiInputBase-adornedStart": {
-        marginLeft: `-${buttonPadding.medium}px`,
+        paddingLeft: `${12 - buttonPadding.medium}px`,
```

Comment:
The idea here was:
- the AdornmentButtons are a bit bigger than designs specify... they include some extra padding to make interacting easier.
- So when there are adornment buttons, add a bit less padding. Originally I was using negative margin, but I had it pon the wrong element.